octodns_edgedns/__init__.py

```diff
@@ -70,18 +70,36 @@ def record_create(self, zone, name, record_type, content):
 
         return result
 
+    def changelist_record_create(self, zone, content):
```

Comment:
I think it'd be cleaner to have a single `record_create` method and then do an `if self.use_changelist:` in that method to decide how to go about things. Same goes for all the other methods here. That'll avoid spreading the logic of `use_changelist` all throughout the code and instead push it down to a single point for each action.